### PR TITLE
Add POST and PUT examples in role bindings API doc

### PR DESCRIPTION
**Pull request by hillaryfraley**
Add POST /rolebindings and PUT /rolebindings/:rolebinding examples.

Epic #1452

### CODE REVIEW

File: content/sensu-go/5.16/api/role-bindings.md [hunk 1]
```diff
@@ -28,7 +28,8 @@ The `/rolebindings` API endpoint provides HTTP GET access to [role binding][1] d
 The following example demonstrates a request to the `/rolebindings` API endpoint, resulting in a JSON array that contains [role binding definitions][1].
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/core/v2/namespaces/default/rolebindings -H "Authorization: Bearer $SENSU_TOKEN"
+curl -X GET http://127.0.0.1:8080/api/core/v2/namespaces/default/rolebindings
```
Comment (Blue0ctober) [inline content/sensu-go/5.16/api/role-bindings.md]: requires ' \ ' at the end of the line, probably should move 'curl -X GET' to it's own line for consistency

File: content/sensu-go/5.16/api/role-bindings.md [hunk 2]
```diff
@@ -122,7 +140,8 @@ The `/rolebindings/:rolebinding` API endpoint provides HTTP GET access to [role
 In the following example, querying the `/rolebindings/:rolebinding` API endpoint returns a JSON map that contains the requested [`:rolebinding` definition][1] (in this example, for the `:rolebinding` named `readers-group-binding`).
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/core/v2/namespaces/default/rolebindings/readers-group-binding -H "Authorization: Bearer $SENSU_TOKEN"
+curl -X GET http://127.0.0.1:8080/api/core/v2/namespaces/default/rolebindings/readers-group-binding
```
Comment (Blue0ctober) [inline content/sensu-go/5.16/api/role-bindings.md]: requires ' \ ' at the end of the line, probably should move 'curl -X GET' to it's own line for consistency

**Review by Blue0ctober — CHANGES_REQUESTED** [review]
See comments for recommended changes

File: content/sensu-go/5.16/api/role-bindings.md (hunk 1 above)
Comment (hillaryfraley) [inline content/sensu-go/5.16/api/role-bindings.md]:
```suggestion
curl -X GET \
http://127.0.0.1:8080/api/core/v2/namespaces/default/rolebindings \
```

File: content/sensu-go/5.16/api/role-bindings.md (hunk 2 above)
Comment (hillaryfraley) [inline content/sensu-go/5.16/api/role-bindings.md]:
```suggestion
curl -X GET \
http://127.0.0.1:8080/api/core/v2/namespaces/default/rolebindings/readers-group-binding \
```

**Pull request merged**